src/auditlog/mixins.py

```diff
@@ -39,26 +49,61 @@ def resource_url(self, obj):
     resource_url.allow_tags = True
     resource_url.short_description = 'Resource'
 
+    def remote_addr_url_w(self, obj):
+        if obj.remote_addr is None:
+            return None
+        link = "https://ipinfo.io/" + str(obj.remote_addr)
```

Comment:
I don't like relying on a remote service that might not exist later on or have been poisoned. Imagine if ipinfo.io goes bankrupt and a malware site opens up, now all people clicking on an auditlog link are being directed to a malware site.

I think we should probably drop this from this PR for now and search for a better implementation. Perhaps an option to hook up to ipinfo's API if the developer wants to set that up and make it so through a switch remote addr url lookup info can be added and possibly made into a celery task so it doesn't slow down the response cycle